pipestat_example/looper_config_pipestat.yaml

```diff
@@ -0,0 +1,32 @@
+pep_config: ./project/project_config_pipestat.yaml # pephub registry path or local path
+output_dir: "./results"
+pipeline_interfaces:
+  sample:  ./pipeline/pipeline_interface.yaml
+  project: ./pipeline/pipeline_interface_project.yaml
+pipestat:
+  project:
+    project_name: test
+    sample_name: frog_1
+    schema_path: ./pipeline/pipestat_output_schema.yaml
+    results_file_path: "results.yaml"  #this is an absolute path for now
+    database:
```

Comment:
these database config options should not need to be duplicated

Reply:
also, I wonder if the simple hello-world example should just use a file back-end. or, maybe we should have both examples.

Reply:
I chose to go with a simple filebackend example.